x-pack/plugins/enterprise_search/public/applications/workplace_search/components/layout/account_header/account_header.test.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { setMockValues } from '../../../../__mocks__';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { EuiHeader, EuiPopover } from '@elastic/eui';
+
+import { AccountHeader } from './';
+
+describe('AccountHeader', () => {
+  const mockValues = {
+    account: {
+      isAdmin: true,
+    },
+  };
+
+  beforeEach(() => {
+    setMockValues({ ...mockValues });
+  });
+
+  it('renders', () => {
+    const wrapper = shallow(<AccountHeader />);
+
+    expect(wrapper.find(EuiHeader)).toHaveLength(1);
+  });
+
+  describe('accountSubNav', () => {
+    it('handles popover trigger click', () => {
+      const wrapper = shallow(<AccountHeader />);
+      const popover = wrapper.find(EuiPopover);
+      popover.dive().find('[data-test-subj="AccountButton"]').prop('onClick')!({} as any);
+
+      expect(popover.prop('isOpen')).toEqual(false);
```

Comment:
Shouldn't `isOpen` become `true` when we click on AccountButton?

Reply:
It should and nothing I did locally seemed to change that. IMO this test is unnecessary since EUI components themselves are tested. I refactored to a test that gives 100% coverage and does not give an incorrect value 🤷